.github/workflows/test.yml

```diff
@@ -12,8 +12,32 @@ env:
   PY_COLORS: 1
 
 jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version:
+          - '3.7'
```

Comment:
Python 3.7 is EOL per 2023-06-27. Should we drop it? :thinking:

Reply:
Probably :thinking: But preferrably in a separate PR.